### ISSUE TITLE
Changes required for getting syndesis working on CRC

### ISSUE THREAD

**Issue by phantomjinx** [syndesisio/syndesis#6918]
Placeholder to log issues with installation and execution of syndesis on [CRC](https://github.com/code-ready/crc).

Log issues as comments and include `crc version` information.

**Comment by phantomjinx** [comment]
```
$ crc version
version: 1.0.0-rc.0+34371d3
OpenShift version: 4.2.0-0.nightly-2019-09-26-192831 (embedded in binary)
```
- Encountered error with oc version checks in syndesis scripts (corrected in [PR-6919](https://github.com/syndesisio/syndesis/pull/6919))
- Postgresql 9.5 is not available from crc by default so upgrade to 9.6 ([PR-6919](https://github.com/syndesisio/syndesis/pull/6919))
- Found 1st install failed with Postgresql 9.6 reporting "insufficient memory"
  - Limit / requirement set at 512MB
  - Lowered limit to 100MB and installed correctly
  - Logged [issue](https://github.com/code-ready/crc/issues/706) & followed suggestion to increase memory of CRC from 12GB to 16GB which was successful in running all syndesis pods
  - With Postgresql being able to run on only 8GB, it remains to be seen whether this was a one-off or whether such steep memory limits are required for syndesis (shouldn't be!)
- Cannot login with developer account & unable therefore to route to syndesis application

**Comment by phantomjinx** [comment]
Duplicate of #6905
